data/spells/scripts/support/find_person.lua

```diff
@@ -46,6 +46,7 @@ function onCastSpell(creature, variant)
 	end
 
 	local playerPosition = creature:getPosition()
+	local targetPosition = targetPlayer:getPosition()
```

Comment:
This should not matter, if the text variable of variant is nil, it will return false.

https://github.com/WibbenZ/forgottenserver/blob/062fe5f2b0403e4786ffd01a60dd29e94906ef81/data/spells/scripts/support/find_person.lua#L37-L40